Source/SwiftLintFramework/Extensions/NSRegularExpression+SwiftLint.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  NSRegularExpression+SwiftLint.swift
+//  SwiftLint
+//
+//  Created by Scott Hoyt on 1/21/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+
+public extension NSRegularExpression {
+    public convenience init(pattern: String) throws {
+        try self.init(pattern: pattern,
+            options: [.AnchorsMatchLines, .DotMatchesLineSeparators])
+    }
+    public static func forcePattern(pattern: String) -> NSRegularExpression {
```

Comment:
Yeah, I was planning on dealing with this all in a branch dedicated to optimizing regex usage as talked about in #394. I can make it internal, but then I will need to `@testable import SwiftLintFramework` in the tests, and it seemed like we might be staying away from that?